corehq/apps/style/management/commands/update_manifest.py

```diff
@@ -18,10 +18,15 @@ class ResourceCompressError(Exception):
 
 class Command(BaseCommand):
     help = "Prints the paths of all the static files"
-    args = "save or soft"
 
     root_dir = settings.FILEPATH
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'params',
```

Comment:
Is there a way to define this or that in the parser? Or could a help text be added here to say the only valid things are save or soft?